README.md

```diff
@@ -67,8 +67,9 @@ app.post('/blogs', cattleguard(config, client), (req, res, next) => {
     default: `false`
   * `perMethod`: boolean, true if rate limiting should happen per HTTP method.
     default: `false`
-  * `lookup`: function, used to return an identifier for the user requesting.
-    This will have the effect of creating a per-user rate limit.
+  * `lookup`: function, used to return an identifier (or array of identifier
```

Comment:
Can you update this comment in the index docblock and add an example of doing this in the code snippet above?